spec/rental_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../rental'
+require_relative '../person'
+require_relative '../book'
+
+describe Rental do
+  before :all do
+    @book = Book.new('Abc', 'Xyz')
+    @student = Student.new(14, 6, 'Neira', parent_permission: false)
+    @rental = Rental.new('19/01/05', @student, @book)
+  end
+  context 'Testing rental Class' do
+    it 'Should have date 19/01/05' do
+      date = @rental.date
+      expect(date).to eql '19/01/05'
+    end
+
+    it 'Should have person named Neira' do
+      person_name = @rental.person.name
+      expect(person_name).to eql 'Neira'
+    end
+
+    it 'Should have book named Abc' do
+      book_title = @rental.book.title
+      expect(book_title).to eql 'Abc'
+    end
+  end
+end
```

Comment:
- [ ] [REQUIRED] Hello Team, great job creating unit tests for all the methods in all the classes, however, the `rental_spec.rb` test fails. The test looks good, but I do not really know why it fails.  Let's ensure we fix this to get this project approved. 
![Screenshot from 2022-05-25 22-06-22](https://user-images.githubusercontent.com/58692590/170368075-711e7e3a-bee2-44c8-af8d-c95ace5c2be3.png)

Reply:
Can you kindly run it again? It is working fine on the device of all the partners.